src/.vuepress/theme/components/BannerTop.vue

```diff
@@ -155,13 +155,15 @@ $topBannerHeightMobile ?= 3.125rem
 // Adjust titles margin and padding for anchor links
 .theme-container.has-top-banner
   {$contentClass}:not(.custom)
-    h1, h2, h3, h4, h5, h6
-      margin-top (0.5rem - $topBannerHeight - $navbarHeight)
-      padding-top ($navbarHeight + $topBannerHeight + 1rem)
+    :not(.style-example)
```

Comment:
I'm not sure this fix is quite correct. This change will also impact other headings because the `h1`, etc. are usually direct children of the `.theme-default-content`, so there's no element for the `:not(.style-example)` to match.

Reply:
Hi @skirtles-code, thanks for the description, I can confirm the issue. There's an upcoming banner copy update, so a new PR will be submitted today. I will address this issue there.

Reply:
@skirtles-code here's the PR that updates the copy and solves this issue: https://github.com/vuejs/docs/pull/1108